src/containers/AddCompanies/container.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import AddCompanies from 'components/AddCompanies';
+import actions from 'store/features/showCompanies/actions';
+
+const mapDispatchToProps = dispatch => ({
+	addCompany: company => dispatch(actions.addCompany(company)),
+});
```

Comment:
Another thing, if you're actionCreator function, in this case `actions.addCompany` takes an argument, you have to make sure you are adding those parameters to the dispatch that's wrapping the function, this stumped me for a while!